user-guide.md

```diff
@@ -54,16 +54,16 @@ and, currently, [decorator](https://github.com/micheles/decorator) as well,
 which can be installed via `pip`:
 
 ```shell
-pip install fabric decorator
+pip install .
 ```
 
 Running the script the first time using the option `--add_platform` queries the
 user for a few infomation about the platform e.g. username and path to the
 private ssh-key, and stores these in a configuration file at
-`.config/platforms/platforms.ini` for later use:
+`JupyterDaskOnSLURM/config/platforms/platforms.ini` for later use:
 
 ```shell
-python runJupyterDaskOnSLURM.py --add_platform
+python src/runJupyterDaskOnSLURM.py --add_platform
```

Comment:
Also here, one more level should be added.

Reply:
the command `python src/runJupyterDaskOnSLURM.py` is now replaced by `jupyter-dask-on-slurm`.